pkg/kubectl/describe.go

```diff
@@ -775,6 +786,27 @@ func describeContainers(pod *api.Pod, out io.Writer) {
 	}
 }
 
+func describeProbe(probe *api.Probe) string {
```

Comment:
If you are not going to refactor `describeContainers` by 1) making it work w/o accepting a pod, 2) export it, then we should at least export this helper.